mgmtsystem_nonconformity/security/mgmtsystem_nonconformity_security.xml

```diff
@@ -8,7 +8,7 @@
             <field name="name">mgmtsystem_nonconformity multi-company</field>
             <field name="model_id" ref="model_mgmtsystem_nonconformity"/>
             <field name="global" eval="True"/>
-            <field name="domain_force">['|',('company_id','=',False),('company_id','child_of',[user.company_id.id])]</field>
+            <field name="domain_force">['|', ('company_id', '=', False), ('company_id', 'in', company_ids)]</field>
```

Comment:
Are you sure this it shouldn't be `company_ids.ids`?

Reply:
Yes, I followed [this guide](https://www.odoo.com/documentation/13.0/howtos/company.html#security-rules) and in my develop environment works fine